MontyHallKata/Models/DoorsFactory.cs

```diff
@@ -0,0 +1,51 @@
+using MontyHallKata.Models.Entity;
+
+namespace MontyHallKata.Models
+{
+    public static class DoorsFactory
+    {
+        public static Door CreateWinningDoor()
+        {
+            var winningDoor = new Door
+            {
+                IsWinningDoor = true
+            };
+            return winningDoor;
+        }
+        public static Door CreateLosingDoor()
+        {
+            var winningDoor = new Door
+            {
+                IsWinningDoor = false
+            };
+            return winningDoor;
+        }
+
+        public static void OpenDoor(Door door)
+        {
+            door.IsOpen = true;
+        }
+        public static bool IsDoorOpen(Door door)
+        {
+            return door.IsOpen;
+        }
+
+        public static void SelectDoor(Door door)
+        {
+            door.IsSelected = true;
+        }
+        public static void DeSelectDoor(Door door)
+        {
+            door.IsSelected = false;
+        }
```

Comment:
This is getting *really* nitpicky, but just be mindful of keeping your spacing consistent between functions. Note how some don't have a gap between and some do.